src/Afsy/Bundle/FrontBundle/Resources/views/Avent/day_2017_21.html.twig

```diff
@@ -526,7 +526,7 @@ private function sendMessageToChannel(ConnectionInterface $conn, $channel, $user
 
   <ul>
     <li>mettre à jour le système d’envoi de messages au <em lang="en">backend</em></li>
-    <li>
+    <li>{# -#}
```

Comment:
the goal of this change is to avoid this alignment issue due to having leading spaces in some items only:

![afsy_websocket](https://user-images.githubusercontent.com/439401/34379344-1cc17474-eafb-11e7-91c0-6798468c4443.png)

I decided to keep the formatting on a newline here (we could have remove the spaces entirely in the template).
As there is no Twig block at this point, I used a comment to hold the whitespace control (which will remove the following spaces during the template compilation).